whatis277/harvest
# bean: Add basic separated route handlers

Separates route handlers

Testing instructions: 
1. dc up bean
2. Visit [/](http://localhost:8080/)
3. Go to [/get-started](http://localhost:8080/get-started)
4. Enter an email and "login"
5. Ensure you get a message indicating an email was sent
